pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -374,7 +374,7 @@ func (p *PriorityQueue) isPodWorthRequeuing(logger klog.Logger, pInfo *framework
 	if !ok {
 		// shouldn't reach here unless bug.
 		logger.Error(nil, "No QueueingHintMap is registered for this profile", "profile", pInfo.Pod.Spec.SchedulerName, "pod", klog.KObj(pInfo.Pod))
-		return framework.QueueAfterBackoff
+		return framework.QueueSkip
```

Comment:
semantically, if no hint registered, the scheduling will never success, so `QueueSkip` reads more reasonable, although it will not happen.

Reply:
Well... 
I'd like to keep it `QueueAfterBackoff` so that if there is a bug in the scheduler and someone reaches here, if we return Skip, that Pod will never be enqueued. 
So, returning AfterBackoff here is kind of safe-guard.

What do you think